tests/baselines/reference/autoAccessor7(target=es2022,usedefineforclassfields=false).js

```diff
@@ -0,0 +1,29 @@
+//// [autoAccessor7.ts]
+abstract class C1 {
+    abstract accessor a: any;
+}
+
+class C2 extends C1 {
+    accessor a = 1;
+}
+
+class C3 extends C1 {
+    get a() { return 1; }
+}
+
+
+//// [autoAccessor7.js]
+class C1 {
+}
+class C2 extends C1 {
+    constructor() {
+        super(...arguments);
+        this.#a_accessor_storage = 1;

```

Comment:
does the emit of `#a_accessor_storage` matter in this case? Could it be the same as the emit for usedefineforclassfields=true?

Reply:
If you are using `Set` instead of `Define` and have other public fields, they all need to be moved to the constructor:

```ts
// ts
class C2 extends C1 {
  x = 1;
  accessor a = 2;
  y = 3;
}

// js, useDefineForClassFields: true
class C2 extends C1 {
  x = 1;
  #a_accessor_storage = 2;
  y = 3;
  get a() { return this.#a_accessor_storage; }
  set a(value) { this.#a_accessor_storage = value; }
}

// js, useDefineForClassFields: false
class C2 extends C1 {
  constructor() {
    super(...arguments);
    this.x = 1;
    this.#a_accessor_storage = 2;
    this.y = 3;
  }
  #a_accessor_storage;
  get a() { return this.#a_accessor_storage; }
  set a(value) { this.#a_accessor_storage = value; }
}
```

Simplifying the emit for the corner case of `useDefineForClassFields: false` when there are no other non-private fields seems like overkill.